spec/requests/preferences/preference_spec.rb

```diff
@@ -81,4 +81,60 @@
       end
     end
   end
+
+  describe 'DELETE destroy' do
+    subject { delete preference_path(preference) }
+
+    let!(:user) { create(:user) }
+    let!(:preference) { create(:preference, user_id: user.id) }
+
+    context 'when logged in' do
+      before do
+        sign_in user
+      end
+
+      it 'destroys the preference' do
+        expect {
+          delete preference_path(preference)
+        }.to change(Preference, :count).by(-1)
+      end
+
+      it 'redirects to the preferences path with a success notice' do
+        subject
+        expect(response).to redirect_to(preferences_path)
+        expect(flash[:notice]).to eq(I18n.t('views.preferences.destroy_success'))
```

Comment:
In this kind of expectation we could test the exact text string here `Preference successfully removed.`
This way if we accidentally made a mistake on the string the test will catch it.

Reply:
That makes sense. I will implement it!